code/game/turfs/simulated/minerals.dm

```diff
@@ -132,7 +141,7 @@
 
 /turf/closed/mineral/random
 	var/mineralSpawnChanceList
-		//Currently, Adamantine won't spawn as it has no uses. -Durandan
+		//Currently, Adamantine won't spawn as it has no uses. -Durandan // HAHA FUCK YOU IT DOES NOW BITCHES
```

Comment:
okay but only if I can find a timestamped one from urist for comparison and replacement